test/Microsoft.AspNet.Mvc.FunctionalTests/JsonOutputFormatterTests.cs

```diff
@@ -72,7 +73,21 @@ public class JsonOutputFormatterTests
 
             // Assert
             Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
-            Assert.Equal(expectedOutput, await response.Content.ReadAsStringAsync());
+
+            var actualContent = await response.Content.ReadAsStringAsync();
+            Assert.Equal(expectedOutput, actualContent);
+
+
+            var modelStateErrors = JsonConvert.DeserializeObject<Dictionary<string, string[]>>(actualContent);
```

Comment:
If you try to deserialize into a `SerializableError` directly, you end up with `JArray[]` which seems consistent with what we used to do.